db/migrate/20180209163253_split_post_text_into_replies.rb

```diff
@@ -0,0 +1,50 @@
+class SplitPostTextIntoReplies < ActiveRecord::Migration[5.1]
+  def up
+    Post.find_each do |post|
+      post.replies.ordered.each { |reply| reply.update_columns(reply_order: reply.reply_order+1) }
```

Comment:
This is going to be super slow. Uhhh, since this all goes on in a transaction, put something like this outside the loop above? (I'd appreciate it being tested in practice :P)

```rb
execute <<-SQL
UPDATE replies
SET reply_order = reply_order + 1;
SQL
```

Reply:
I was kinda assuming someone was gonna do SQL Magic to it before it was released yeah